### ISSUE TITLE
Color contrast issues on /marketing/traffic examples

### ISSUE THREAD

**Issue by youbrokesomething** [Automattic/wp-calypso#75599]
### Quick summary

The example links on /marketing/traffic are too light blue for the grey background that they're on. This happens in a few different places, so a default color somewhere may need to be updated to a darker color or this particular instance may need a CSS override.

The text links are in the parent element:
`<div class="related-posts__preview-items">`

### Steps to reproduce

1. On /marketing/traffic
2. Inspect the link texts in the post example and manually run the foreground and background colors through a color contrast checker or use an accessibility checker like the axe DevTools chrome browser extension.

### What you expected to happen

All text on the page should have a color contrast of at least 4.5:1 at a normal font size.

### What actually happened

Three links have just a shade too light text color. We can darken the color by a single shade or make the text bigger.

### Impact

Some (< 50%)

### Available workarounds?

No but the platform is still usable
